### PR TITLE
1528 html example DataTable

### PR DESCRIPTION
**Pull request by ArKhmeln**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/example/complex/DataTableExampleTests.java  1
- jdi-light-html-tests/src/main/java/io/github/com/entities/TableData.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4840068&bid=16071408)

File: jdi-light-html-tests/src/main/java/io/github/com/entities/TableData.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package io.github.com.entities;
+
+public class TableData {
```
Comment (codacy-review) [inline jdi-light-html-tests/src/main/java/io/github/com/entities/TableData.java]: Codacy Issue found: [The utility class name 'TableData' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4840068)

**Comment by mf-code** [comment]
[4510] - https://transfer.sh/zNSiq/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4510] - https://transfer.sh/o70rj/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4510] - https://transfer.sh/lOvxI/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4510] - Allure report on Netlify: https://5e21a8e3974a85006403e677--musing-swartz-6fc060.netlify.com

**Pull request merged**